packages/manager/modules/vps/src/dashboard/rebuild/vps-rebuild.controller.js

```diff
@@ -9,13 +9,26 @@ export default class VpsRebuildController {
   $onInit() {
     this.vpsOptions = {
       doNotSendPassword: false,
+      installRTM: true,
```

Comment:
If you set `installRTM` to `false` here, you could remove the test `options.installRTM && this.rtmAvailable()` into `rebuildVps` call, and just set `installRTM: options.installRTM`, line L30 no ?

Reply:
checkbox need to be checked by default